claas/eval/README.md

```
@@ -0,0 +1,70 @@
+    --num-steps 10
+```
+
+Or as a module:
+
+```bash
+python -m claas.eval --preferences no_emoji --metrics all --num-steps 20
+```
+
+Results are written to `--output-dir` (default `./eval_results`). View them in the browser:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)